### ISSUE TITLE
Quantum theory interpretations

### ISSUE THREAD

**Issue by SM-3000** [SolemnShark871/YAItQC#28]
I'm not sure how useful it is to discuss interpretations of quantum theory when we can just stick with mathematical models that reflect empirical observations that are relevant to the foundations of quantum computing.

The main problem with talking about trends in interpretations of quantum theory is that to give an accurate view, we'd have to summarise and cite whatever primary and extremely-trustworthy secondary sources say on this. If there's clusters of similar views, we would have to talk about the nature of the clusters and fairly determine what those clusters are (one of which may be the things that are, sometimes individually and sometimes collectively, often called "(the) Copenhagen interpretation").

I'm guessing that you don't want even just the intro to become even a summary of a history of quantum theory textbook. If I'm wrong, then I can help you more with this while you focus on the more hands-on QC stuff.

**Comment by SolemnShark871** [comment]
This is a very good point, I'd agree with saying very little and linking a few good references for further reading. Ideally just a couple of sentences for each interpretation explaining what it is and maybe some proi's & con's of each

**Comment by SM-3000** [comment]
That sounds good. It'll take me at least a week to cover this very fairly, but if my plans don't change, I should be able to do something. Would you like me to work on this?

**Comment by SolemnShark871** [comment]
I'd appreciate that a lot, thank you! 

P.S. if you check out the acknowledgements section you'll find a little something

**Comment by SM-3000** [comment]
https://www3.nd.edu/~dhoward1/Howard%20Who%20Invented%20the%20Copenhagen%20Interpretation.pdf So far I've found this, where there's an attempt to distance the Copenhagen interpretation from Bohr and attribute it only to Heisenberg. The writer also claims that philosophers of science, (especially Popper), had misattributed it to Bohr, who had a somewhat different interpretation.

**Comment by SolemnShark871** [comment]
I wasn't aware of this, the history of the development of quantum theory would be a very fascinating book to write. If we're just going for a short paragraph this detail could be added without being too verbose

**Comment by SM-3000** [comment]
I have been getting the impression that a lot of the controversies in quantum theory interpretations are hybrids of empirical and broad metaphysical critiques. I think I made a correct guess that people who, as I see it, are today known for their philosophies of science more than any experimental work and close and detailed reviews of it, were very much involved in the debates.

I think one of my other related guesses about politics being involved, sometimes brawls between ideological camps and sometimes grudges between intellectuals. I'm in the middle of reading a paper about Popper's influence - https://arxiv.org/ftp/arxiv/papers/1811/1811.00902.pdf - and it's easy to see examples of both, with Feyerabend having an axe to grind against Popper and Popper himself having his big political and philosophical project, which may both rest on a similar ethical foundation that could've developed from his political experiences - https://www.unav.edu/web/ciencia-razon-y-fe/the-ethical-roots-of-karl-poppers-epistemology

All of that will make given an accurate view way more complicated than I think your document will allow. I may have to do what Dirac once said he'd do and promote a kind of silence on the issue.

That said, I may have found something useful on Wikipedia. I would have to make some corrections and find proper citations for some of these. https://en.wikipedia.org/wiki/Interpretations_of_quantum_mechanics#Comparisons

**Comment by SM-3000** [comment]
Would you like me to write the section itself and save you some work, or just give you notes so that you can build the chapter with your writing style in mind and only including the parts that are more important to the overall book?

**Comment by SolemnShark871** [comment]
Sorry for taking so long to get back to you. I agree with your point that this topic is beyond the scope of what can be properly covered in this book. If you could send me some useful notes & perhaps later we could work together on writing something separate from the book. In a few years, when the main body is more complete, we could add a summary in an appendix. What are your thoughts on this plan?

**Comment by SM-3000** [comment]
I agree. We can make a summary for now and include as many accurate sources as we can. We can just build a short bibliography full of primary literature on the topic and give some brief explanations of the camps in the debate and their claims.

If you want to write something about the horrid philosophical mess later, we can do that if and when you're burned out and you've got nothing else to do. I can afford to make it more of a focus for myself and just give you something to work with. I don't want to speculate on how we'll make this work in future given that our plans may change drastically.
